docs/src/components/Playground/renderConfig.ts

```diff
@@ -0,0 +1,25 @@
+import * as _ from 'lodash'
+import * as React from 'react'
+import * as Stardust from '@stardust-ui/react'
+
+export const babelConfig = {
+  plugins: ['proposal-class-properties', ['transform-typescript', { isTSX: true }]],
```

Comment:
It's strange, but `typescript`'s preset is registered, but not working 🤔 
Will fill an issue to Babel's repo.

Reply:
So, are we evaluating now the examples as js code?

Reply:
yes

Reply:
Nope, all TS transforms are applied. I mentioned that use there the plugin:

```
plugins: [['transform-typescript', { isTSX: true }]],
```

Instead of the preset:

```
presets: ['typescript'],
```

However, **everything** is works.